build/love/frontend/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Clone repo
+FROM alpine/git as cloner
+
+WORKDIR /usr/src/love
+ARG love_frontend
+RUN git clone --branch ${love_frontend} --depth 1 --single-branch https://github.com/lsst-ts/LOVE-frontend.git
+
+# Build artifacts
+FROM node:12.16.1-alpine as builder
+COPY --from=cloner /usr/src/love/LOVE-frontend /usr/src/love/LOVE-frontend
```

Comment:
Missed this one earlier. Falls in the same vein as the comments about the manager Dockerfiles.